torch/package/package_exporter.py

```diff
@@ -44,20 +44,16 @@ class PackageExporter:
     for further code dependencies (`dependencies=True`). It looks for import statements,
     resolves relative references to qualified module names, and calls :method:`require_module`
     on each it finds, recursively resolving dependencies.
-
     """
 
-    importers: List[Callable[[str], Any]]
-    """ A list of functions that will be called in order to find the module assocated
-    with module names referenced by other modules or by pickled objects. Initialized to
-    `[importlib.import_module]` by default. When pickling code or objects that was loaded
-    from an imported packaged, that `importer.import_module` should be put into the importer list.
-    When a name conflict occurs between importers, the first importer in the list takes precedence,
-    and only objects that refer to this first importers class can be saved
+    """A module environment that will be searched in order to find the module assocated with module names
+    referenced by other modules or by pickled objects. The default module environment just uses DefaultImporter,
+    which searches the Python environment. When pickling code or objects loaded from an imported package,
+    you should create a OrderedImporter that has that `PackageImporter` listed first.
     """
+    importer: Importer
 
-
-    def __init__(self, f: Union[str, Path, BinaryIO], verbose: bool = True):
+    def __init__(self, f: Union[str, Path, BinaryIO], importer: Importer = SysImporter, verbose: bool = True):
```

Comment:
It would be nice to allow `importer` to be `Either[Importer, List[Importer]]` and use it to initialize a OrderedImporter, since this will be a common case and prevent having to import the OrderedImporter object in user code.